services/src/pro/datasets/postgres.rs

```diff
@@ -332,6 +332,8 @@ where
         Ok(match meta_data {
             MetaDataDefinition::GdalMetaDataRegular(m) => Box::new(m),
             MetaDataDefinition::GdalStatic(m) => Box::new(m),
+            MetaDataDefinition::GdalMetaDataList(m) => Box::new(m),
+            MetaDataDefinition::GdalMetadataNetCdfCf(m) => Box::new(m),
```

Comment:
Could you add a test that would fail if I'd create a second PR that just removes these lines?

Reply:
https://github.com/geo-engine/geoengine/pull/781/commits/01e46e0041668b6d6a1ceffb856b80b1eedb107c